tasmota/tasmota_xx2c_global/xsns_interface.ino

```diff
@@ -528,8 +528,13 @@ bool (* const xsns_func_ptr[])(uint32_t) = {  // Sensor Function Pointers for si
 #endif
 
 #ifdef XSNS_127
-  &Xsns127
+  &Xsns127,
 #endif
+
+#ifdef XSNS_128
+  &Xsns128
+#endif
+
```

Comment:
DO NEVER CHANGE ANYTHING IN xsns_interface.ino !!!

This will break sensor enable/disable.

As suggested, use the next free slot.